Dockerfile

```diff
@@ -1,33 +1,17 @@
 FROM jenkinsci/jenkins:latest
```

Comment:
and the jenkinsci one - not the "official" one? (ie the one in the root namespace?)